Content/NPCs/Moonstone/Dreambeast.cs

```diff
@@ -634,6 +637,14 @@ public override void Load()
 			On_Main.GUIBarsDraw += DrawLunacyMeter;
 		}
 
+		public override void SendClientChanges(ModPlayer clientPlayer)
```

Comment:
this would send a packet literally every frame and cause severe lag. make sure you perform checks when sending packets to make sure you actually need them

Reply:
also I don't really recommend hooking off of this particular one to begin with you're better off putting network logic where it's more relevant